source/module_hamilt_general/module_surchem/test/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+install(DIRECTORY support DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
+
+list(APPEND depend_files 
+  ../../../module_base/global_variable.cpp
+)
+AddTest(
+  TARGET surchem_atom_in
+  SOURCES atom_in_test.cpp
+)
+
+AddTest(
+  TARGET surchem_cal_epsilon  LIBS ${math_libs} planewave device base
```

Comment:
```suggestion
  TARGET surchem_cal_epsilon
  LIBS ${math_libs} planewave device

Reply:
Perhaps the "base" item is missing.  Should be "LIBS ${math_libs} planewave device base"?

Reply:
Yes, please.